README.md

```diff
@@ -363,8 +363,8 @@ velero install \
     --plugins velero/velero-plugin-for-microsoft-azure:v1.5.0 \
     --bucket $BLOB_CONTAINER \
     --secret-file ./credentials-velero \
-    --backup-location-config resourceGroup=$AZURE_BACKUP_RESOURCE_GROUP,storageAccount=$AZURE_STORAGE_ACCOUNT_ID[,subscriptionId=$AZURE_BACKUP_SUBSCRIPTION_ID] \
```

Comment:
@msdolbey Thanks for the contribution.

The parameters in `[]` mean that they are optional

Reply:
Omg - thank you very much for pointing this out @ywk253100 

Mind if I add a note that these brackets mean that they are optional, or is that just a very well known thing and not necessary?

Reply:
Yes, please add a note for these brackets

Reply:
@ywk253100 thanks for the guidance - updated here